cmd/frontend/graphqlbackend/search.go

```diff
@@ -41,6 +41,17 @@ type SearchArgs struct {
 	First          *int32
 	VersionContext *string
 
+	// Stream if non-nil will send all results down c.
+	//
```

Comment:
```suggestion
```

Reply:
hmm, I still want to say what it is. Will do this instead.
```suggestion
	// Stream if non-nil will stream all SearchEvents.
	//
```